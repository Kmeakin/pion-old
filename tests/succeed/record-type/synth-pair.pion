//~ run = "${PION} elab ${FILE}"

{x : Int, y : Int}
