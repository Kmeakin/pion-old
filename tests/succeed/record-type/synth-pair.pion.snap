exit status: 0
stdout:
{ x : Int, y : Int } : Type

stderr:
